pallets/nfts/src/lib.rs

```diff
@@ -148,7 +148,7 @@ pub mod pallet {
 		/// the `create_collection_with_id` function. However, if the `Incrementable` trait
 		/// implementation has an incremental order, the `create_collection_with_id` function
 		/// should not be used as it can claim a value in the ID sequence.
-		type CollectionId: Member + Parameter + MaxEncodedLen + Copy + Incrementable;
+		type CollectionId: Member + Parameter + MaxEncodedLen + Clone + Incrementable;
```

Comment:
Remove `Copy`, add `Clone`. `xcm::v4::Location` type is not implemented with `Copy`.